apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/doedshendelse/kontrollpunkt/DoedshendelseKontrollpunktService.kt

```diff
@@ -77,4 +81,37 @@ class DoedshendelseKontrollpunktService(
 
         return null
     }
+
+    private fun kontrollerSamtidigDoedsfall(
+        avdoed: PersonDTO,
+        hendelse: Doedshendelse,
+    ): DoedshendelseKontrollpunkt? =
+        try {
+            avdoed.doedsdato?.verdi?.let { avdoedDoedsdato ->
+                val annenForelderFnr =
+                    pdlTjenesterKlient.hentPdlModell(
+                        foedselsnummer = hendelse.beroertFnr,
+                        rolle = PersonRolle.BARN,
+                        saktype = SakType.BARNEPENSJON,
+                    ).familieRelasjon?.verdi?.foreldre
+                        ?.map { it.value }
+                        ?.firstOrNull { it != hendelse.avdoedFnr }
+
+                if (annenForelderFnr == null) {
```

Comment:
Føler meg ganske keen på å teste dette både happy-day and not

Reply:
Mulig det er flere scenarier her, men denne er testet i https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/3869/files#diff-6fe5628441de220bee0960ee030b427639af97789f5aa04e1727dda29a3e3b2bR179 for tilfeller hvor vi ikke finner annen forelder, og happy-case er testet ved at det ikke opprettes kontrollpunkt for det.